providers/provider-pojo/src/main/java/io/servicecomb/provider/pojo/Invoker.java

```diff
@@ -54,8 +55,8 @@ public Invoker(String microserviceName, String schemaId, Class<?> consumerIntf)
         this.consumerIntf = consumerIntf;
     }
 
-    public void prepare() {
-        referenceConfig = CseContext.getInstance().getConsumerProviderManager().getReferenceConfig(microserviceName);
+    protected void prepare() {
```

Comment:
It's not binary compatible change,  it's better to keep the method signature there.